build/VSL.Imports.Closed.targets

```diff
@@ -70,8 +70,8 @@
           Inputs="@(IntermediateAssembly)"
           Outputs="@(IntermediateAssembly);$(PostCompileBinaryModificationSentinelFile)">
     
-    <Exec Command="$(VSLOutDir)\FakeSign.exe &quot;@(IntermediateAssembly)&quot;" />
-    
+    <Exec Condition=" '$(OS)' == 'Windows_NT'" Command="$(VSLOutDir)\FakeSign.exe &quot;@(IntermediateAssembly)&quot;" />
+    <Exec Condition=" '$(OS)' != 'Windows_NT'" Command="mono $(VSLOutDir)\FakeSign.exe &quot;@(IntermediateAssembly)&quot;" />
```

Comment:
(I am unsure)
Would it make sense to pursue it here: https://github.com/Microsoft/msbuild/blob/82177a50da735cc0443ac10fa490d69368403d71/src/XMakeTasks/Exec.cs?

( curiously, there is [`cmd.exe`](https://github.com/Microsoft/msbuild/blob/xplat/src/XMakeTasks/Exec.cs#L666) in xplat branch as well! :open_mouth: )